cmd/podman/secrets/list.go

```diff
@@ -76,9 +84,21 @@ func ls(cmd *cobra.Command, args []string) error {
 			Driver:    response.Spec.Driver.Name,
 		})
 	}
+
+	if listFlag.quiet && !cmd.Flags().Changed("format") {
```

Comment:
Should we make format and quiet conflict?

Reply:
They don't conflict for other commands (e.g., `image list`) but I think it makes sense to make them conflict for new commands.

Reply:
I'm okay with making them conflict, but Docker allows both to be used and prioritizes format over quiet. Let me know what you think

Reply:
Docker compat wins :)